docs/core/deploying/trim-self-contained.md

```diff
@@ -73,8 +73,8 @@ If we're using .Net 5.0, we can update our project file to include the following
 
 Trim your application using the [dotnet publish](../tools/dotnet-publish.md) command. When you publish your app, set the following properties:
 
-- Publish as a self-contained app for a specific runtime: `-r win-x64`
-- Enable trimming: `/p:PublishTrimmed=true`
+- Publish as a self-contained app for a specific runtime: `<RuntimeIdentifier>win-x64</RuntimeIdentifier>`
```

Comment:
Generally, we've been telling people not to hard-code RIDs into their project file, and instead pass them on the command line / publish profile.

cc @dsplaisted 

Are we telling people to put the RID in the project now because without it, `PublishTrimmed` breaks `dotnet build`? If yes, maybe we should move the validation to the `publish` command that ensures you are publishing SelfContained if `PublishTrimmed=true`.

Reply:
No, `PublishTrimmed` doesn't break build - I was just changing this for consistency with the line below to address https://github.com/dotnet/docs/pull/24740#discussion_r655498139.  I'll change it back to recommending `-r win-x64` and `<PublishTrimmed>true</PublishTrimmed>` but being more explicit about what should go in the project file.